.github/workflows/downstream.yml

```diff
@@ -81,3 +81,21 @@ jobs:
         run: pip install git+https://github.com/astropy/astropy@${{ github.event.inputs.astropy_ref || 'master' }}#egg=astropy[test]
       - name: Run astropy tests
         run: pytest --pyargs astropy
+
+  weldx:
+    name: weldx@master unit tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install asdf
+        run: pip install git+https://github.com/asdf-format/asdf@${{ github.event.inputs.asdf_ref || 'master' }}
+      - name: Install weldx
+        run: |
+          git clone https://github.com/BAMWelDX/weldx.git
+          cd weldx
+          pip install -e .[test]
+      - name: Run weldx tests
```

Comment:
This failed due to missing tests/asdf_tests, it must be that each step resets the working directory before it runs?  Up above we're using `actions/checkout@v2` to fetch the repository which seems to take care of the working directory automatically.

Reply:
Actually now that I say that I see that the weldx steps almost fit into the matrix job.  Maybe we should add a new variable to the matrix for additional pytest arguments?